processing/src/main/java/org/apache/druid/math/expr/ExprEval.java

```diff
@@ -357,17 +360,17 @@ public static ExprEval ofArray(ExpressionType outputType, @Nullable Object[] val
    * instead.
    */
   @Deprecated
```

Comment:
is this method really `@Deprecated` ? doesn't looks like it..

Reply:
ideally nothing new should use it, it is for the legacy boolean handling mode (strictBooleans = false)